packages/react/src/components/MultiSelect/FilterableMultiSelect.tsx

```diff
@@ -1213,5 +1215,3 @@ FilterableMultiSelect.propTypes = {
    */
   warnText: PropTypes.node,
 };
-
-export default FilterableMultiSelect;
```

Comment:
Removing this exposes a slight risk of breaking someone who might be using the default export with a fully qualified path, like:

```
import FilterableMultiSelect from '@carbon/react/es/components/MultiSelect/FilterableMultiSelect'
```

Reply:
Is it advisable for a consumer to be using a fully qualified path? The component docs say to import from `@carbon/react` (https://github.com/carbon-design-system/carbon/tree/main/packages/react#usage) and the first option that intellisense gives most of the time is to import from `@carbon/react`.

Just wondering why someone would import from `@carbon/react/es/components/MultiSelect/FilterableMultiSelect` when (1) nothing suggests that they should and (2) it's much simpler to import from `@carbon/react`. I can revert the change if you would like. Just trying to understand whether we should be supporting something that I can't find documentation for when the obvsious alternative is documented.

Reply:
> Do you know why someone would import from @carbon/react/es/components/MultiSelect/FilterableMultiSelect when (1) nothing suggests that they should and (2) it's much simpler to import from @carbon/react?

We run into [Hyrum's Law](https://www.hyrumslaw.com/) a lot. I've seen folks try to sidestep treeshaking issues in their environments or "manually" optimize builds by using fully qualified paths.

Reply:
I'm cool to merge with this and see how it shakes out. If it turns into a noisy break we'll have to revert this PR and issue a patch. Not a huge deal, just a heads up.